contributing.md

```diff
@@ -18,10 +20,17 @@ Example for .NET:
 
 ```
 $ git clone git@github.com:username/azure-activedirectory-library-for-dotnet.git
-$ cd azure-activedirectory-library-for-dotnet
+$ rename azure-activedirectory-library-for-dotnet adal.net
 $ git remote add upstream git@github.com:AzureAD/azure-activedirectory-library-for-dotnet.git
 ```
 
+> Important node: 
+ADAL.Net uses a multi-targeting projects to generate assemblies for several platforms. There are issues with the Nuget package manager in Visual Studio 2017, not recogizing nuget pacakges in the ADAL.NET project. Therefore you'd want to restore the nuget packages using this command line:
+
+```
+$ msbuild ADAL.NET.NoWinRT.sln /t:restore 
+```
+
 Now decide if you want your feature or bug fix to go into the current stable version or the next version of the library. 
```

Comment:
we should add a blurb about how to run unit tests from command line as the tests do no show up in test explorer sometimes.